sparse/coo/core.py

```diff
@@ -1436,6 +1436,22 @@ def __rmatmul__(self, other):
         except NotImplementedError:
             return NotImplemented
 
+    def __array_function__(self, func, types, args, kwargs):
```

Comment:
I think it's worth moving this to `SparseArray`, and it'll be inherited automatically.

Reply:
Yes, thanks for that, I totally missed the inheritance over there.